src/core/client/admin/ui/components/Table/TableBody.css

```diff
@@ -0,0 +1,2 @@
+.root {
```

Comment:
Are we keeping this so custom CSS can style against it? Or can we delete this file?

Reply:
yeah i left it for future use because that seems to be a pattern but i'm ok with deleting it

Reply:
No worries, if you're seeing it elsewhere, then I'm fine with this too. I usually delete empty classes, but this isn't going to break any camel's backs.